### PR TITLE
Feature/add item

### PR DESCRIPTION
**Pull request by rudik-byte**

### CODE REVIEW

**Review by olehkril — CHANGES_REQUESTED** [review]

File: src/main/java/com/global/inbox/controller/TestController.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.global.inbox.controller;
```
[the remark this answers is not in the document]
Comment (rudik-byte) [reply src/main/java/com/global/inbox/controller/TestController.java:1]: done

File: src/main/java/com/global/inbox/model/Item.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
+package com.global.inbox.model;
```
[the remark this answers is not in the document]
Comment (rudik-byte) [reply src/main/java/com/global/inbox/model/Item.java:1]: What do you mean?

File: src/main/java/com/global/inbox/model/ItemStatus.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package com.global.inbox.model;
```
[the remark this answers is not in the document]
Comment (rudik-byte) [reply src/main/java/com/global/inbox/model/ItemStatus.java:1]: What do you mean?

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-web'
```
Comment (olehkril) [inline build.gradle:22]: Marked as done, but you need to push your changes

**Review by olehkril — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	//spring
+	implementation group: 'org.springframework', name: 'spring-webflux', version: '5.3.21'
```
Comment (olehkril) [inline build.gradle:23]: If you have starter, then you don't need spring-webflux seprately.

File: build.gradle [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+	implementation group: 'org.springframework.boot', name: 'spring-boot-starter-webflux', version: '2.7.1'
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa:2.7.1'
+	//test
```
Comment (olehkril) [inline build.gradle:26]: test to the bottom

File: build.gradle [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+	testImplementation 'org.springframework.boot:spring-boot-starter-test'
+	//database
+	implementation 'org.liquibase:liquibase-core'
+	runtimeOnly 'org.postgresql:postgresql'
+	//lombok
```
Comment (olehkril) [inline build.gradle:31]: split in sections like following

```
//spring
compile()
compile()

//test
testCompile()
```

File: src/main/java/com/global/inbox/controller/ItemsController.java [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/items")
+public class ItemsController {
+    @GetMapping
```
Comment (olehkril) [inline src/main/java/com/global/inbox/controller/ItemsController.java:12]: new line

File: DevOps-local/docker-compose.yaml [hunk 9]
```diff
@@ -0,0 +1,11 @@
+version: '3.7'
```
Comment (olehkril) [inline DevOps-local/docker-compose.yaml:1]: DevOps-local > environment

Comment (olehkril) [reply DevOps-local/docker-compose.yaml:1]: Also, you can add some small instruction into `README.md` file. 

## Local usage...

### Prerequisites 

1. Docker
2. Docker compose

In order to run locally... 

`cd environment && docker compose up -d `

**Review by olehkril — CHANGES_REQUESTED** [review]

**Review by olehkril — APPROVED** [review]

**Pull request merged**